DZ_1_OOP.py

```diff
@@ -0,0 +1,32 @@
+# ДЗ_1 - Жила Сергій. Добрий день!
+
+class Product:
+    def __init__(self, name_product: str, price: float):
+        self.name_product = name_product
```

Comment:
Тут трохи проблемний неймінг. Якщо у тебе не має іншого атрибути, що відповідає за назву чогось ще, то можна не уточнювати, що це назва товару.
```suggestion
        self.name= name_product
```